src/app/verify/mfa/index.njk

```diff
@@ -30,29 +30,19 @@
         {% include "common/templates/includes/success-action.njk" %}
       {% endif %}
 
-      <h1 class="govuk-heading-l">
-        Confirm your multi-factor authentication code
-      </h1>
+      <h1 class="govuk-heading-l">{{__('title_confirm_mfa_code')}}</h1>
 
-      <p class="govuk-body-l">
-        If the email that you entered is associated with an account then you will have received
-        an email with your login code. Please enter it below.
-      </p>
+      <p class="govuk-body-l">{{__('confirm_mfa_code_caption')}}</p>
 
       <form action="/login/authenticate" method="post">
-        <div class="govuk-form-group {{ m.form_group_class('mfa-authentication-code', errors) }}">
```

Comment:
Just changed from "mfa-authentication-code" to "mfa-code" just for the sake of less typing